kitctl/main.go

```diff
@@ -0,0 +1,382 @@
+/*************************************************************************
+ * Copyright 2021 Gravwell, Inc. All rights reserved.
+ * Contact: <legal@gravwell.io>
+ *
+ * This software may be modified and distributed under the terms of the
+ * BSD 2-clause license. See the LICENSE file for details.
+ **************************************************************************/
+
+package main
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/gravwell/gravwell/v3/client/types"
+	"github.com/gravwell/gravwell/v3/client/types/kits"
+	"github.com/gravwell/gravwell/v3/ingesters/utils"
+)
+
+func usage() {
+}
+
+func main() {
+	flag.Parse()
+
+	args := flag.Args()
+	if len(args) == 0 {
+		log.Printf("Must specify a command.")
+		usage()
+		return
+	}
+
+	switch args[0] {
+	case "unpack":
+		// Unpack the given kit into the current directory
+		unpackKit(args[1:])
+	case "pack":
+		// Pack the current directory into a kit
+		packKit(args[1:])
+	case "info":
+		// Information about the kit in the current directory
```

Comment:
can we throw an error here about "no ready yet" rather than silently do nothing.  Just in case someone tries to use the dev branch.